src/core/import-maps.js

```diff
@@ -0,0 +1,22 @@
+// @ts-check
+// Temporary workaround until browsers get real import-maps
```

Comment:
So much sadness... but ok :)

Reply:
What's going to be the long term strategy so we don't need this?

Reply:
Official import maps implementation can replace this with traditional `import x as "module"`.